src/server/AuctionHouseServer/AuctionHouse.API/DTO/UserDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+namespace AuctionHouse.API.DTO
+{
+    public class UserDto
+    {
+        private string _name;
+        private string _password;
+        private string _email;
+        private string _dateOfBirth;
+        public UserDto(UserCredentials userCredentials)
+        {
+            _name = userCredentials.Username;
+            _password = userCredentials.Password;
+            _email = userCredentials.Email;
+            _dateOfBirth = userCredentials.Date;
+        }
+
+        public string getEmail()
+        {
+            return _email;
+        }
```

Comment:
As long as it's dto there is no need to hide properties behind hand written getters